_posts/2019-01-10-running-an-appveyor-build-after-creating-an-azure-web-app-using-terraform.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Running an AppVeyor Build After Creating an Azure Web App Using Terraform
+date: 2019-01-10 15:00:00
+tags:
+- Terraform
+- OSS
+categories:
+- development
+twitter_text: "Running an AppVeyor Build After Creating an Azure Web App Using Terraform"
+authors: Ken Dale
+image: https://images.unsplash.com/photo-1487958449943-2429e8be8625?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2100&q=80
+image_url: https://unsplash.com/photos/QdAAasrZhdk
+image_credit: Lance Anderson
+---
+
+We're in the process of scaffolding out our [Microsoft Azure](https://azure.microsoft.com) environments using [Terraform](https://www.terraform.io/). While Terraform does an excellent job creating Azure Web Apps, it still needs code to be deployed before the application can be used.
+
+We currently manage our deployments using AppVeyor. Here's how to get the latest of `master` deployed after the resource is created:
```

Comment:
The line "Here's how to get the latest..." sounds vague to me. Maybe "Here's how to get terraform to queue a build..." or something like that.